src/model/commands/accounts.rs

```diff
@@ -0,0 +1,213 @@
+/// The purpose of add signatory command is to add an identifier to the account. Such
+/// identifier is a public key of another device or a public key of another user.
+#[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct AddSignatory {
+    pub account_id: String,
+    pub public_key: [u8; 32],
+}
+
+/// # Example
+/// ```
+/// use iroha::model::commands::accounts::AddSignatory;
+///
+/// let command_payload = AddSignatory {
+///     account_id: "account@domain".to_string(),
+///     public_key: [63; 32],
+/// };
+/// let result: Vec<u8> = command_payload.into();
+/// ```
+impl std::convert::From<AddSignatory> for Vec<u8> {
+    fn from(command_payload: AddSignatory) -> Self {
```

Comment:
It's probably not a question for this specific PR, but still. Why do we take ownership of `command_payload` here and in other conversions. Isn't it better to take reference, so we won't need to clone?

To my mind we would not encounter any problem with references here.

Reply:
https://soramitsu.atlassian.net/browse/RH2-23